xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-pageobjects/src/main/java/org/xwiki/ckeditor/test/po/RichTextAreaElement.java

```diff
@@ -155,4 +156,18 @@ private WebElement getActiveElement()
         getDriver().switchTo().frame(this.iframe);
         return getDriver().switchTo().activeElement();
     }
+    
+    /**
+     * Waits until a macro appears in the document.
+     */
+    public void waitUntilMacroAppears()
+    {
+        try {
+            getDriver().waitUntilElementDisappears(this.iframe, By.tagName("iframe"));;
```

Comment:
I don't understand this line. You pass the iframe as the parent and you're looking for a child element with name "iframe"? Doesn't make sense, because the iframe element doesn't have child nodes. There's also a double ``;`` at the end.

Reply:
Thank you, I'm not even sure how the test can pass despite that mistake

Reply:
Correction, the test fails when I remove the parent, I'm confused

Reply:
It passes because it waits for the iframe to not have a child with tag `iframe`, a condition that is instantly true.